Berksfile

```diff
@@ -4,5 +4,9 @@ metadata
 group :test do
   cookbook 'java'
```

Comment:
this is where you should pin java to ~> 8.4, not in the cookbook's metadata. I don't want to force a specific method of java installation on anyone, so I don't want the cookbook to rely on the java cookbook directly.